lib/workers/repository/init/index.ts

```diff
@@ -48,5 +48,6 @@ export async function initRepo(
       'Full resolved config and hostRules including presets'
     );
   }
+  await cloneSubmodules(!!config.cloneSubmodules);
```

Comment:
This means that repos with cloneSubmodules will have 100% git cloning (even if no work needs to be done), but we can live with that edge case.